### ISSUE TITLE
maybe parsing error of md5 file

### ISSUE THREAD

**Issue by MartinHaimberger** [rhash/RHash#235]
Hi while checking my file, i came over some files which resulted in an error. It looks like a error in parsing/reading the md5 checksum file.

The pars of the checksum file which did not work correctly.
```
...
a29a34b26687829858c30b74f7b8b786  7eventy 5ive (2007)\7eventy 5ive (2007)-fanart.jpg
4b2d5e570f34387d65334616639a75fa  7eventy 5ive (2007)\7eventy 5ive (2007).jpg
20c8a4b8ed350a62a0a8e3344da36af7  7eventy 5ive (2007)\7eventy 5ive (2007).mkv
7b25aaa9d6b20125eeb29ff8c55676b7  7eventy 5ive (2007)\7eventy 5ive (2007).nfo
4b2d5e570f34387d65334616639a75fa  7eventy 5ive (2007)\7eventy 5ive (2007).tbn
...
```

Which lead to the result:
```
...
5ive (2007)/7eventy 5ive (2007)-fanart.jpg          No such file or directory
5ive (2007)/7eventy 5ive (2007).jpg                 No such file or directory
5ive (2007)/7eventy 5ive (2007).mkv                 No such file or directory
5ive (2007)/7eventy 5ive (2007).nfo                 No such file or directory
5ive (2007)/7eventy 5ive (2007).tbn                 No such file or directory
...
```
It seems that the parsing of the direcotry / filename has a problem here.
The md5 file as created with the newest rhash on windows and read and check on linux.

**Comment by therube** [comment]
What command line did you use to create the hashes?
What command line did you use to check the hashes?

The opening "7eventy 5ive (2007)\" is a directory?

Your results do not show the "7eventy " part, only starting with the "5ive" part.
Also the slashes are reversed (from \ to /).

```
C:\TMP\BRU>rhash --md5 -r 7e*
d41d8cd98f00b204e9800998ecf8427e  7eventy 5ive (2007)\7eventy 5ive (2007)-fanart
.jpg
d41d8cd98f00b204e9800998ecf8427e  7eventy 5ive (2007)\7eventy 5ive (2007).mkv
d41d8cd98f00b204e9800998ecf8427e  7eventy 5ive (2007)\7eventy 5ive (2007).nfo

C:\TMP\BRU>rhash --md5 -r 7e* >777

C:\TMP\BRU>rhash -c 777

--( Verifying 777 )-------------------------------------------------------------

7eventy 5ive (2007)\7eventy 5ive (2007)-fanart.jpg  OK
7eventy 5ive (2007)\7eventy 5ive (2007).mkv         OK
7eventy 5ive (2007)\7eventy 5ive (2007).nfo         OK
--------------------------------------------------------------------------------

Everything OK
```

**Comment by MartinHaimberger** [comment]
Yes 7eventy 5ive (2007) is a directory.

Created the file on windows (RHash v1.4.3) used this command line:
```
rhash -rM --percents --speed . > directory.md5
```
copied the file over to linux (RHash v1.4.1) (that may explain the reversed slashes)
and used the following comandline in the directory containing the direcotry.md5 file
```
rhash -cr -M --crc-accept=md5 . | tee result.txt
```

this gives me the error on the files in this directory with the odd (partially missing) directory name. All other directories and files got checked ok.

**Comment by rhash** [comment]
Duplicate of #139 and #148, fixed in v1.4.2.
